parm/config/gfs/config.landanl

```diff
@@ -1,23 +1,35 @@
 #! /usr/bin/env bash
 
 ########## config.landanl ##########
-# configuration common to all land analysis tasks
+# configuration common to land analysis tasks
 
 echo "BEGIN: config.landanl"
 
+# Get task specific resources
+. "${EXPDIR}/config.resources" landanl
+
 obs_list_name=gdas_land_adpsfc_only.yaml
 if [[ "${cyc}" == "18" ]]; then
     obs_list_name=gdas_land_prototype.yaml
 fi
 
 export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/config/
 export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/lists/${obs_list_name}
-export LANDVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/land/letkfoi/letkfoi.yaml
-export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+
+# Name of the JEDI executable and its yaml template
+export JEDIEXE="${HOMEgfs}/exec/fv3jedi_letkf.x"
+export JEDIYAML="${HOMEgfs}/sorc/gdas.cd/parm/land/letkfoi/letkfoi.yaml"
+
+# Ensemble member properties
+export SNOWDEPTHVAR="snwdph"  # if FRACGRID=NO, else "snodl"
+export BESTDDEV="30."  # Background Error Std. Dev. for LETKFOI
+
+# Name of the executable that applies increment to bkg and its namelist template
+export FRACGRID="yes"
```

Comment:
I don't know.  @jiaruidong2017 and @CoryMartin-NOAA may have a plan for this.

Reply:
I agree with @CoryMartin-NOAA because there are many `frac_grid` found from GFS physics model. In the namelist of `apply_incr_nml`, the `frac_grid` is assigned with `$FRACGRID` (defined as yes/no). With the changes to the variable `frac_grid` from yes/no to Fortran boolean made by @aerorahul for the `apply_jedi_incr` code at PR https://github.com/NOAA-PSL/land-apply_jedi_incr/pull/3, we can replace the `$FRACGRID` with `$FRAC_GRID` (defined at `config.base`). We will double check with @barlage and @ClaraDraper-NOAA

Reply:
defining `FRAC_GRID` is done in `config.base`.  This instance is now removed from `config.landanl`.